middlewares/validateUser.js

```diff
@@ -0,0 +1,26 @@
+import { ObjectId } from 'mongodb';
+
+import db from './../db.js';
+
+export async function validateUser (req,res,next) {
+
+  try {
+
+    const { token } = req.params;
```

Comment:
Aí, se alterar a requisição para pegar o token pelo header, precisa alterar aqui para parar de pegar pelo parâmetro. Se for alterar isso mesmo, tem um outro middleware que está fazendo a mesma validação pelo header e respondendo a mesma informação do usuário (validateSession.js).